lib/containers/container_images.pm

```diff
@@ -258,17 +258,15 @@ sub test_zypper_on_container {
     assert_script_run("$runtime run $image zypper lr -s", 120);
 
     if ($runtime =~ /buildah/) {
-        # zypper ref
-        assert_script_run("$runtime run $image -- zypper -v ref | grep \"All repositories have been refreshed\"", 120);
+        assert_script_run("$runtime run $image -- zypper -n ref", 120);
 
         # Create new image and remove the working container
         assert_script_run("$runtime commit --rm $image refreshed", 120);
 
         # Verify the new image works
-        assert_script_run("$runtime run \$($runtime from refreshed) -- zypper -v ref | grep \"All repositories have been refreshed\" ", 120);
```

Comment:
i dont see any reason to remove the verbose mode. Could you add it back?

Reply:
done